i18n/en.json

```diff
@@ -2323,6 +2323,10 @@
     "id": "api.post.search_posts.invalid_body.app_error",
     "translation": "Unable to parse the request body."
   },
+  {
+    "id": "api.post.search_posts.invalid_body.exclusive_parameters",
+    "translation": "Request contains mutually exclusive parameters."
```

Comment:
I know this is a bad error message. Any suggestions?

Reply:
I think you could mention the impacted parameters in the error message as well so they know what to correct

Reply:
Yeah, I'd accept a slice and print them as part of the error.